GenderPayGap.WebUI/Views/ActionHubOld/Promising.cshtml

```diff
@@ -0,0 +1,189 @@
+﻿@using GenderPayGap.WebUI.Models.Shared
+@using GenderPayGap.WebUI.Views.Components.PaginationLinks
+@{
+    ViewBag.Title = "Actions to close the gap - Gender pay gap service";
+    Layout = "Index.cshtml";
+    var index = 1;
+}
+
+<h2 class="govuk-heading-l">Promising actions</h2>
+<p class="govuk-body">
+    These actions are promising and require further research to improve the evidence on their effectiveness and how best to implement them.
+    The government has policies in place supporting some of these actions to improve women's participation in the workforce.
+    The government is evaluating their effectiveness and we recommend that employers evaluate their actions too.
+</p>
+
+<div class="action-step">
+    <i class="icon icon-step-@(index++)"></i>
+    <h3 class="govuk-heading-s">Improve workplace flexibility for men and women</h3>
+    <p class="govuk-body">
+        <strong class="govuk-tag">
+          Government Initiative
+        </strong>
+    </p>
+    <ul class="govuk-list govuk-list--bullet">
+        <li>Advertise and offer all jobs as having flexible working options, such as part-time work, remote working, job sharing or compressed hours</li>
+        <li>Allow people to work flexibly, where possible</li>
+        <li>Encourage senior leaders to role model working flexibly and to champion flexible working</li>
+        <li>Encourage men to work flexibly, so that it isn’t seen as only a female benefit</li>
+    </ul>
+</div>
+
+<div class="action-step">
+    <i class="icon icon-step-@(index++)"></i>
+    <h3 class="govuk-heading-s">Encourage the uptake of Shared Parental Leave</h3>
+    <p class="govuk-body">
+            <strong class="govuk-tag">
+              Government Initiative
+            </strong>
+        </p>
+    <p class="govuk-body">
+        The gender pay gap widens dramatically after women have children but this could be reduced if men and women were able to share childcare more equally.
+        Shared Parental Leave and Pay enables working parents to share up to 50 weeks of leave and up to 37 weeks of pay in their child’s first year.
+    </p>
+    <ul class="govuk-list govuk-list--bullet">
+        <li>Offer enhanced Shared Parental Pay at the same level as enhanced maternity pay</li>
+        <li>
+            Encourage take up of Shared Parental Leave (see our <a target="_blank" rel="noopener" href="https://sharedparentalleave.campaign.gov.uk/"><span class="visually-hidden">Shared Parental Leave </span>guidance<span class="visually-hidden">(opens in a new window)</span></a>). For example:
+            <ul class="govuk-list govuk-list--bullet">

```

Comment:
Agreed - design-wise I was a bit unsure what to do here because the old version used dots for top-level list items, and open circles for the second-level items. I could recreate that with some custom CSS, but given that the design system uses the same list bullet for all levels as default I thought saving the code and keeping it consistent was good enough rationale to not bother?